src/extension/background-script/actions/accounts/export.ts

```diff
@@ -0,0 +1,26 @@
+import { decryptData } from "~/common/lib/crypto";
+import state from "~/extension/background-script/state";
+import type { MessageAccountEdit } from "~/types";
+
+const exportAccount = async (message: MessageAccountEdit) => {
```

Comment:
The type is wrong. Would be good to add a new type and create a test like we do for other account-actions that use TS already.

Reply:
Ah you created a type but don't use it here :)

Reply:
Oops forgot to change :P

Reply:
Could you add a test for this as well, like we have for the other ones?:  
https://github.com/getAlby/lightning-browser-extension/tree/master/src/extension/background-script/actions/accounts/__tests__

Reply:
Yup! Currently working on the same locally, will push it soon!